services/blockchain-connector/shared/sdk/blocks.js

```diff
@@ -24,31 +24,31 @@ const {
 
 const getLastBlockDecoded = async () => {
 	const block = await getLastBlock();
-	const decodedBlock = decodeBlock(block);
+	const decodedBlock = formatBlock(block);
 	return decodedBlock;
```

Comment:
Could we also please refactor the variables everywhere to be relevant?
Eg: Use `formattedBlock` instead of `decodedBlock`.
```suggestion
	const formattedBlock = formatBlock(block);
	return formattedBlock;
```